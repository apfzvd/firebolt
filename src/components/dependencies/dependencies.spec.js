import { shallow } from 'enzyme'
import React from 'react'
import Dependencies from './index'
/** @test {Dependencies} */
const wrapper = shallow(<Dependencies />)

describe('dependecies', () => {
  it('snapshot counter not connect', () => {
    expect(wrapper.debug()).toMatchSnapshot()
  })
})
