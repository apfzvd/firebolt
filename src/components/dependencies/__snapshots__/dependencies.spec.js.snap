// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`dependecies snapshot counter not connect 1`] = `
"<ul className={[undefined]}>
  <div className={[undefined]}>
    <Panel className={[undefined]} brand={{...}} title={false} size=\\"medium\\" loading={false} accordion={false} open={true}>
      <a href=\\"https://www.cypress.io/\\">
        <img width={50} height={50} src={{...}} alt=\\"Logo Redux\\" />
      </a>
      <span>
        Redux
      </span>
    </Panel>
    <Panel className={[undefined]} brand={{...}} title={false} size=\\"medium\\" loading={false} accordion={false} open={true}>
      <a href=\\"https://github.com/b2wads/grimorio-ui\\">
        <img width={50} height={50} src={{...}} alt=\\"Logo Eslint\\" />
      </a>
      <span>
        Eslint
      </span>
    </Panel>
    <Panel className={[undefined]} brand={{...}} title={false} size=\\"medium\\" loading={false} accordion={false} open={true}>
      <a href=\\"https://github.com/immerjs/immer\\">
        <img width={50} height={50} src={{...}} alt=\\"Logo Prettier\\" />
      </a>
      <span>
        Prettier
      </span>
    </Panel>
  </div>
  <div className={[undefined]}>
    <Panel className={[undefined]} brand={{...}} title={false} size=\\"medium\\" loading={false} accordion={false} open={true}>
      <a href=\\"https://www.cypress.io/\\">
        <img width={50} height={50} src={{...}} alt=\\"Logo Jest\\" />
      </a>
      <span>
        Jest
      </span>
    </Panel>
    <Panel className={[undefined]} brand={{...}} title={false} size=\\"medium\\" loading={false} accordion={false} open={true}>
      <a href=\\"https://github.com/b2wads/grimorio-ui\\">
        <img width={50} height={50} src={{...}} alt=\\"Logo Stylus\\" />
      </a>
      <span>
        Stylus
      </span>
    </Panel>
    <Panel className={[undefined]} brand={{...}} title={false} size=\\"medium\\" loading={false} accordion={false} open={true}>
      <a href=\\"https://github.com/immerjs/immer\\">
        <img width={50} height={50} src={{...}} alt=\\"Logo React-router-dom\\" />
      </a>
      <span>
        React-router-dom
      </span>
    </Panel>
  </div>
  <div className={[undefined]}>
    <Panel className={[undefined]} brand={{...}} title={false} size=\\"medium\\" loading={false} accordion={false} open={true}>
      <a href=\\"https://www.cypress.io/\\">
        <img width={50} height={50} src={{...}} alt=\\"Logo Cypress\\" />
      </a>
      <span>
        Cypress
      </span>
    </Panel>
    <Panel className={[undefined]} brand={{...}} title={false} size=\\"medium\\" loading={false} accordion={false} open={true}>
      <a href=\\"https://github.com/b2wads/grimorio-ui\\">
        <img width={50} height={50} src={{...}} alt=\\"Logo Grimorio\\" />
      </a>
      <span>
        Grimorio
      </span>
    </Panel>
    <Panel className={[undefined]} brand={{...}} title={false} size=\\"medium\\" loading={false} accordion={false} open={true}>
      <a href=\\"https://github.com/immerjs/immer\\">
        <img width={50} height={50} src={{...}} alt=\\"Logo Immer\\" />
      </a>
      <span>
        Immer
      </span>
    </Panel>
  </div>
</ul>"
`;
